packages/ui/src/components/va-navbar/VaNavbar.vue

```diff
@@ -173,9 +165,15 @@ export default defineComponent({
   }
 
   @include media-breakpoint-down(sm) {
-    height: top-mobile-nav-height;
+    flex-direction: column;
+    align-items: center;
+    height: $top-mobile-nav-height;
```

Comment:
We've decided not to use global variables, because we're planning to get rid of them over time.